graphql/cxs-impl/src/main/java/org/apache/unomi/graphql/types/output/CDPProfileConnection.java

```diff
@@ -40,10 +40,14 @@ public CDPProfileConnection(List<CDPProfileEdge> edges, CDPPageInfo pageInfo) {
         this.pageInfo = pageInfo;
     }
 
+    @GraphQLField
+    @GraphQLName("edges")
```

Comment:
Use accessors without get instead of @GraphQLName in entire class plz to be uniform across the app